include/Yash.h

```diff
@@ -74,11 +74,10 @@ class Yash {
             print("\r\n");
             if (!m_command.empty()) {
                 runCommand();
-                m_commandHistory.push_back(m_command);
-
-                if (m_commandHistory.size() > m_commandHistorySize)
+                if (m_commandHistory.size() >= m_commandHistorySize)
```

Comment:
Probably should handle the case of size == 0 here :) Seems like it's allowed to not have a history

Reply:
Good idea :+1: